pkg/helm/handlers/handlers.go

```diff
@@ -240,3 +253,31 @@ func (h *helmHandlers) HandleGetReleaseHistory(user *auth.User, w http.ResponseW
 	w.Header().Set("Content-Type", "application/json")
 	w.Write(res)
 }
+
+func (h *helmHandlers) HandleGetRepos(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	conf := h.getActionConfigurations(h.ApiServerHost, "", user.Token, &h.Transport)
+	config, err := conf.RESTClientGetter.ToRESTConfig()
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusInternalServerError, serverutils.ApiError{Err: fmt.Sprintf("Failed to get k8s config: %v", err)})
+		return
+	}
+	client, err := h.getDynamicClient(config)
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusInternalServerError, serverutils.ApiError{Err: fmt.Sprintf("Failed to get k8s dynamic client: %v", err)})
+		return
+	}
+
+	kubeClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusInternalServerError, serverutils.ApiError{Err: fmt.Sprintf("Failed to get k8s core client: %v", err)})
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/yaml")
+	w.Header().Set("Cache-Control", "private, no-cache")
```

Comment:
Note that this won't prevent the browser from caching, just it has to revalidate. If that's intentional, OK :) If you don't want this to ever be cached, use `no-store`.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Fixed this https://github.com/openshift/console/pull/5933/commits/ed5fb99532657ac1251b7fab81c574cd189be54d